frontend/src/pages/member/JoinPage.vue

```diff
@@ -94,9 +94,18 @@ export default {
         return;
       }
       try {
-        // TODO member를 생성하는 API를 추가해주세요.
-        // const { email, age, password } = this.member;
-        // await fetch("/join", { email, age, password })
+        const { email, age, password } = this.member;
+        await fetch("http://localhost:8080/members", {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            email: email,
+            age: age,
+            password: password
```

Comment:
```suggestion
            email,
            age,
            password
```

변수의 이름과 값이 동일할 경우 위와 같이 적용 가능합니다 : )

Reply:
와우!! 감사합니다!! 재연링은 JS도 빠삭하시군요..